glitch/apps.py

```diff
@@ -1,5 +1,6 @@
 import imageio
 import numpy as np
+from random import choice
```

Comment:
i think is better to use the same RNG for everything, so `np.random.choice`. this way is easier to control the randomness, if needed.

Reply:
The `np.random.choice` won't work for selecting an array from inside another array, so we should keep `random.choice`.